### ISSUE TITLE
k8s 1.16 deprecated APIs

### ISSUE THREAD

**Issue by cicdteam** [werf/kubedog#146]
Hi!

Please look at https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

as result 

```
E1010 13:50:35.116136    4072 reflector.go:131] pkg/mod/k8s.io/client-go@v0.0.0-20190411052641-7a6b4715b709/tools/cache/reflector.go:99: Failed to list *v1beta1.Deployment: the server could not find the requested resource
```

when I run `kubedog folllow deployment` over k8s 1.16 cluster

**Comment by distorhead** [comment]
Hi! Do you use latest 0.3.4 release https://github.com/flant/kubedog/releases/tag/v0.3.4 with k8s 1.16 support?

Also follow subcommand is deprecated, better migrate to multitrack: https://github.com/flant/kubedog#multitracker-cli

**Comment by kivagant-ba** [comment]
@distorhead could you please show some examples of how to use `kubedog multitrack` as replacement for `follow` or `rollout`? The `multitrack` and it exits fast and isn't watching for changes and I didn't find arguments for this.

And also the syntax of `multitrack` looks way harder for manual usage.

**Comment by ilya-lesikov** [comment]
This is probably long-fixed or not an issue by now with the 1.29 as the latest Kubernetes release.

@kivagant-ba We will deprecate Kubedog CLI in the future to focus on it as an easily embedded library. For out-of-the-box CLI experience we recommend using [werf](https://github.com/werf/werf). Later we will also release [Nelm](https://github.com/werf/nelm) as a standalone binary as a direct Helm replacement with Kubedog built-in.
